libvirt.go

```diff
@@ -521,6 +521,11 @@ func (d *Driver) getIPByMacFromSettings(mac string) (string, error) {
 	}
 	var s []Lease
 
+	// In case of status file is empty then don't try to unmarshal data
```

Comment:
I think it's slightly better worded as "If status file is empty, then don't try to unmarshal data" or "In case status file is empty, don't try to unmarshal data"